x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -182,7 +182,7 @@ docker.elastic.co/observability/profiling-agent:${stackVersion} /root/pf-host-ag
           }),
           content: (
             <EuiCodeBlock paddingSize="s" isCopyable>
-              {`sudo pf-host-agent/pf-host-agent -project-id=1 -secret-token=${secretToken} -collection-agent=${collectionAgentHost}`}
+              {`cd pf-host-agent-${stackVersion}-* && sudo ./pf-host-agent -project-id=1 -secret-token=${secretToken} -collection-agent=${collectionAgentHost}`}
```

Comment:
The `cd` leaves you in that directory. So why not
```
{`sudo pf-host-agent-${stackVersion}-*/pf-host-agent -project-id=1 -secret-token=${secretToken} -collection-agent=${collectionAgentHost}`}
```

Reply:
Not sure... the ideal solution for be to have dedicated sub-tabs for each architecture, so we could use the full path, and we could avoid using the wildcard.

@cauemarcondes do you know if it's possible to have "nested" tabs, one for each architecture we support?

Reply:
Yes, that is possible, some refactoring is needed for that, but that can be done!

Reply:
Can you point me in the right direction? Would love to get this in for the binary and symbtool instructions, basically everywhere we have a double arch supported.

Reply:
Let me draft something... will be back!!!

Reply:
Thanks a lot for the help on sorting this out 🙏🏼